<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>381</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Pic 10A. Introduction to Programming.</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Pic 10B. Intermediate Programming.</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Pic 10C. Advanced Programming</string>
     </property>
    </item>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>380</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Schema A</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>410</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Schema B</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_course">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Course :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_overallScore">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>440</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Overall Score:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>440</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>321</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Hw 7</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>284</y>
      <width>32</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Hw 6</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>174</y>
      <width>32</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Hw 3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>358</y>
      <width>32</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Hw 8</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>248</y>
      <width>32</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Hw 5</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>395</y>
      <width>32</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Hw 9</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>101</y>
      <width>30</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Hw 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>431</y>
      <width>38</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Hw 10</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>138</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Hw 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>211</y>
      <width>32</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Hw 4</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_10">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>437</y>
      <width>84</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_5">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>252</y>
      <width>84</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_7">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>326</y>
      <width>84</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_9">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>400</y>
      <width>84</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>178</y>
      <width>84</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>104</y>
      <width>84</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_4">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>215</y>
      <width>84</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_6">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>289</y>
      <width>84</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>141</y>
      <width>84</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_8">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>363</y>
      <width>84</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_2">
    <property name="geometry">
     <rect>
      <x>281</x>
      <y>140</y>
      <width>48</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_3">
    <property name="geometry">
     <rect>
      <x>281</x>
      <y>177</y>
      <width>48</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_4">
    <property name="geometry">
     <rect>
      <x>281</x>
      <y>214</y>
      <width>48</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_5">
    <property name="geometry">
     <rect>
      <x>281</x>
      <y>251</y>
      <width>48</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>281</x>
      <y>103</y>
      <width>48</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_6">
    <property name="geometry">
     <rect>
      <x>281</x>
      <y>288</y>
      <width>48</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_7">
    <property name="geometry">
     <rect>
      <x>281</x>
      <y>325</y>
      <width>48</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_9">
    <property name="geometry">
     <rect>
      <x>281</x>
      <y>399</y>
      <width>48</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_10">
    <property name="geometry">
     <rect>
      <x>281</x>
      <y>436</y>
      <width>48</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_8">
    <property name="geometry">
     <rect>
      <x>281</x>
      <y>362</y>
      <width>48</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_13">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>303</y>
      <width>84</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_12">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>210</y>
      <width>84</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_11">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>117</y>
      <width>84</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_11">
    <property name="geometry">
     <rect>
      <x>353</x>
      <y>151</y>
      <width>48</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_13">
    <property name="geometry">
     <rect>
      <x>353</x>
      <y>337</y>
      <width>48</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>353</x>
      <y>92</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Midterm 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>353</x>
      <y>278</y>
      <width>65</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Final Exam</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_12">
    <property name="geometry">
     <rect>
      <x>353</x>
      <y>244</y>
      <width>48</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>353</x>
      <y>185</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Midterm 2</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>575</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
